### ISSUE TITLE
Feature Request: viewForCardAtIndex: and indexForCardWithView:

### ISSUE THREAD

**Issue by efremidze** [Yalantis/Koloda#37]
I was wondering if we could add a few functions to the `KolodaView` to return the `viewForCardAtIndex:` and `indexForCardWithView:`. Only alternative is to retain the views in a separate datasource. I can make a pull request with the feature.

**Comment by AEugene** [comment]
I think it's not good way to retain all shown cards somewhere. Could you describe the purpose of this and then we'll try to figure out the best solution.

**Comment by efremidze** [comment]
I'm doing a transition from the tinder card view into a detail view like the tinder app where i pass the UIImageView from the card to the details view header. I need a reference to the UIImageView, so i'm retaining the top card. `viewForCardAtIndex:` isn't necessary, an alternative is to add a helper method to return the visible cards. Any other suggestions?

**Comment by RayanSaeed** [comment]
@efremidze Hey, did you find any solution I basically want to do something similar as well.. Just need the reference to view for the top card, because I need to transition with it to another view like a flip animation..

**Comment by efremidze** [comment]
A quick solution is adding this method to `KolodaView.swift`:
```
public func viewForTopCard() -> UIView? {
    return visibleCards.first?.contentView
}
```

**Comment by AEugene** [comment]
Fixed with #48

**Comment by efremidze** [comment]
Thanks!!!!!
